sdk/src/cache/cache_client.rs

```diff
@@ -76,12 +77,14 @@ use crate::{utils, IntoBytes, MomentoResult};
 /// ```
 #[derive(Clone, Debug)]
 pub struct CacheClient {
-    pub(crate) data_client: ScsClient<InterceptedService<Channel, HeaderInterceptor>>,
+    pub(crate) data_clients: Vec<ScsClient<InterceptedService<Channel, HeaderInterceptor>>>,
```

Comment:
I realize this is not where pub(crate) started, but it seems more 🤨 with this as a vec property

Reply:
For my benefit, you mean that this should be private and if we need an accessor function we should provide one, as opposed to `pub(crate)` which is read/write, correct?